src/client.ts

```diff
@@ -352,6 +362,23 @@ export class SubscriptionClient {
     }
   }
 
+  private clearInactivityTimeout() {
+    if (this.inactivityTimeoutId) {
+      clearTimeout(this.inactivityTimeoutId);
+      this.inactivityTimeoutId = null;
+    }
+  }
+
+  private setInactivityTimeout() {
```

Comment:
What should the behavior of this function be as it related to `lazy = false` since `unsubscribe` is the only place it gets called.

Reply:
Should this function also test for connection to exist? I think this function is idempotent as is and can be called in any state. Not sure on the impact of `lazy` flag on this? Can you please elaborate.